bedrock/firefox/views.py

```diff
@@ -176,6 +191,33 @@ def sign_attribution_codes(codes):
     return {"attribution_code": code.decode(), "attribution_sig": sig}
 
 
+@require_POST
+@csrf_exempt
```

Comment:
Do we need to make this CSRF exempt? If you want to add CSRF back in, we can get the CSRF token via some boilerplate JS - https://docs.djangoproject.com/en/4.0/ref/csrf/#ajax